src/compiler/types.ts

```diff
@@ -7098,7 +7008,10 @@ export interface ProjectReference {
     path: string;
     /** The path as the user originally wrote it */
     originalPath?: string;
-    /** True if the output of this reference should be prepended to the output of this project. Only valid for --outFile compilations */
+    /**
+     * Only present for deprecate notification

```

Comment:
I _think_ we can declaration merge from the deprecation helper, if that's what you mean. (See also how `src/services/types.ts` extends the compiler.)

Reply:
yeah but what would it solve. we still need to check "ref.prepend" in program.ts and utilities to compare and issue errors?

Reply:
Oh, gotcha.